src/main/java/com/artipie/api/artifactory/GetPermissionSlice.java

```diff
@@ -111,7 +112,22 @@ private static JsonObject response(
         return Json.createObjectBuilder()
             .add("includesPattern", includesPattern(patterns))
             .add("repositories", Json.createArrayBuilder().add(repo).build())
-            .add("principals", Json.createObjectBuilder().add("users", users(permissions)))
+            .add(
+                "principals",
+                Json.createObjectBuilder()
+                    .add(
+                        "users",
+                        permissions(
+                            permissions.stream().filter(item -> item.username().charAt(0) != '/')
+                        )
+                    )
+                    .add(
+                        "groups",
+                        permissions(
+                            permissions.stream().filter(item -> item.username().charAt(0) == '/')
+                        )
```

Comment:
@olenagerasimova I think we need to extract `item.username().charAt(0) == '/'` this predicate to a method or class and reuse it in both filter expressions here to avoid code duplication and make intent more clear